src/ui/component/discoveryFirstRun/view.jsx

```diff
@@ -0,0 +1,57 @@
+// @flow
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+import vis from 'vis';
+import { Form, FormField } from 'component/common/form';
+
+type Props = {
+  onSelect: string => void,
+  selected: Array<string>,
+  selectableTags: Array<string>,
+};
+
+export default function DiscoveryFirstRun(props: Props) {
+  console.log('props', props);
+
+  const { tags, doToggleTag, doDeleteTag } = props;
+  const [newTag, setNewTag] = useState('');
+
+  function onChange(e) {
+    setNewTag(e.target.value);
+  }
+
+  function handleSubmit() {
+    doToggleTag(newTag);
+    setNewTag('');
+  }
+
+  return (
+    <section className="discover-section">
```

Comment:
Maybe `tag-chooser` or `tag-selector` or something like this? 

This component should also be named the same or similar as the outer class name, rather than `DiscoveryFirstRun`.